### ISSUE TITLE
HW performance counters are now available

### ISSUE THREAD

**Issue by mo-tenstorrent** [tenstorrent/tt-metal#13940]
Review https://github.com/tenstorrent/tt-metal/compare/miacim/perf_counters?expand=1

And determine how they can be integrated to profiler.

**Comment by mo-tenstorrent** [comment]
related to https://github.com/tenstorrent/tt-metal/issues/14847

**Comment by mo-tenstorrent** [comment]
Blocked by https://github.com/tenstorrent/tt-metal/issues/14050 to first bring the support

**Comment by pgkeller** [comment]
is this a dupe of #16649 ?

**Comment by mo-tenstorrent** [comment]
No these are performance debug registers. Tensix usage and so on ... The list actually grows even further with Quasar.

**Comment by minaliuTT** [comment]
Per @mo-tenstorrent, this ticket is valid but low priority, and will be reviewed by the team when resources allow.

**Comment by mo-tenstorrent** [comment]
Their fetching is no implemented, now we need to integrate them with tracy GUI.
